tests/data/configs/cms.php

```diff
@@ -32,5 +33,9 @@
             'password' => DB_PASS,
             'charset' => 'utf8',
         ],
+        'cache' => \yii\caching\ArrayCache::class,
+        'assetManager' => [
+            'basePath' => '/var/www/luya/envs/phpunit/public_html/assets'
```

Comment:
@boehsermoe try this:

```
'assetManager' => [
                    'basePath' => dirname(__DIR__) . '/assets',
                ],
```

see here: https://github.com/luyadev/luya-bootstrap4/blob/master/tests/src/BaseBootstrap4BlockTestCase.php